_beats/dev-tools/packer/readme.md.j2

```diff
@@ -1,19 +0,0 @@
-# Welcome to {{.beat_name}} {{.version}}
```

Comment:
missed this on first review.  Removing this will undo the readme fixes from #449